src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,38 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int AGE_MIN_VALUE = 18;
+    public static final int PASSWORD_MIN_LENGTH = 6;
+    private static final StorageDaoImpl storageDaoImpl = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null) {
+            throw new RuntimeException("Fill user information with valid data.");
+        }
+        if (user.getLogin() == null) {
+            throw new RuntimeException("Enter your login." + user.getLogin());
+        }
+        if (user.getAge() == null) {
+            throw new RuntimeException("Enter your age." + user.getAge());
+        }
```

Comment:
```suggestion
        if (user.getLogin() == null) {
            throw new RuntimeException("Login could not be null. User " + user);
        }
        if (user.getAge() == null) {
            throw new RuntimeException("Age could not be null. User " + user);
        }
```

And we should define toString() method in user class (without password field)